src/lib.rs

```diff
@@ -461,8 +467,8 @@ mod tests {
 
     #[test]
     fn multibyte_chars() {
-        assert_eq!(normalize_word("piétro"), "piétro");
-        assert_eq!(normalize_word("piéitly"), "piéatly");
+        assert_eq!(normalize_word("piétro"), "peetro");
+        assert_eq!(normalize_word("piéitly"), "peeetly");
```

Comment:
peeeeeeeeetly :smile:

Reply:
Yeah. I had half a mind to try to fix that. We have a thing that reduces doubles but it leaves ee and oo alone.  But ss and tt and rr all drop down. I think a minor change could make sure triples of `eee` or `ooo` reduce to just `oo` or `ee`.  Maybe I'll give that a try.

Reply:
Well, it's more complicated than I thought. I'm just going to ignore it for now.   At the moment, our sliding lookback is based on the original word and not the modified word. Punting...